icefoganalytics/elcc-data-management
# Add Relative Path Support To Api Typescript

Depends on https://github.com/icefoganalytics/elcc-data-management/pull/17
Fixes https://github.com/icefoganalytics/elcc-data-management/issues/14

# Context

Add support for imports as `@/some/file` instead of `../../../../some/file`
`@/...` now means `api/src` folder.

# Testing Instructions

1. Test the development setup by booting the app via `dev up`.
2. Go to http://localhost:8080 and log in.
3. Check that the app loads content from the api.
4. Test the production setup by booting the app via `dev up db` and `docker compose up --build` (and the other instructions in the README).
5. Clear your cookies and repeat steps 2 and 3.
